Dockerfile.template

```diff
@@ -28,13 +28,17 @@ RUN apk add --update -t build-deps build-base git zlib-dev openssl-dev \
  && rm -rf /usr/src/*
 
 # Mirror Websites
-ENV SITES https://5pi.de
+ENV SITES http://elpissite.weebly.com
 RUN mkdir -p /content/www \
  && cd /content/www \
- && httrack $SITES \
+ && httrack --ext-depth=1 $SITES \
```

Comment:
without ext-depth it won't pull external resources (for instance jquery library)